docs/DeveloperGuide.md

```diff
@@ -338,7 +338,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
 
-* **Mainstream OS**: Windows, Linux, Unix, OS-X
+* **Mainstream OS**: Windows, Linux, Unix, MacOS
+* **CLI**: Command Line Interface
+* **GUI**: Graphical User Interface
+* **JSON**: [JavaScript Object Notation](https://www.json.org/json-en.html)
 * **Private contact detail**: A contact detail that is not meant to be shared with others
```

Comment:
I think this definition is not necessary because our updated user stories don't mention private contact details, and we aren't planning to handle that explicitly atm. What do you think?